fire/cli/niv/_læs_observationer.py

```diff
@@ -121,6 +121,39 @@ def importer_observationer(projektnavn: str) -> pd.DataFrame:
         f"Fandt {n_tabte} tabte punkter blandt {len(observerede_punkter)} observerede punkter."
     )
 
+    # Check journalsiders konsistens:
+
+    # Vi starter med en dict med en tom mængde for hver strækning
+    journalsider = dict((tuple(sorted(p)), set()) for p in zip(fra, til))
+    # ...så føjer vi hovedjournalsidenummeret til for hver observation
+    for i, par in enumerate(zip(fra, til)):
+        strækning = tuple(sorted(par))
+        hovedside = observationer["Journal"][i].split(":")[0]
+        journalsider[strækning].add(hovedside)
+    # ...og advarer hvis der er strækninger med flere hovedjournalsidenumre
+    for strækning in journalsider:
+        if len(journalsider[strækning]) > 1:
+            fire.cli.print(
+                "ADVARSEL - Flere hovedjournalsider til samme strækning:",
```

Comment:
Advarsler kommer generelt på den her form rundt om i koden:
```suggestion
                "ADVARSEL:Flere hovedjournalsider til samme strækning:",
```

Reply:
Formen her skyldes det syntaktiske problem med at have et kolon efter et kolon. Men jeg følger din opfordring til konsistens

Reply:
Ja, det er også lidt grimt. Der bør findes en generel løsning på de her advarsler og fejlmeldinger.